doc/_utils/redirects.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright (c) 2022 Nordic Semiconductor
+Copyright (c) 2022-2023 Nordic Semiconductor
```

Comment:
Is this change intentional?

Reply:
I'm still trying to figure out what to put here because licence check doesn't like also the old version.